macros/src/main/scala/gestalt/macros/Monadless.scala

```diff
@@ -162,19 +163,13 @@ object Transformer {
           }
 
 
-        case Apply(fun, List(v)) if isUnlift(fun.tpe) => Some(v)
+        case Applysss(fun, _, (v :: Nil) :: Nil) if isUnlift(fun.tpe) => Some(v)
```

Comment:
I'd argue in this case, quasiquote like following is better if quasiquote treat `$fun` to match anything, not just identifier (otherwise, user has to write another case `$path.$fun[$tp]($v)`):

```Scala
case q"$fun[$tp]($v)" if isUnlift(fun.tpe) => Some(v)
```